.env

```diff
@@ -74,3 +74,7 @@ ROUTEMASTER_MONITOR_INTERVAL=10
 ROUTEMASTER_SCALING_THRESHOLD=100
 ROUTEMASTER_SCALING_DEADLINE=1000
 
+# maximum number of bytes in an event's optional data payload (when messagepack'ed)
+# DO NOT INCREASE THIS - using the event bus as a data bus is a strong smell of
+# an incompatible architecture, and will cause major bus scalability issues.
+ROUTEMASTER_MAX_EVENT_DATA=64
```

Comment:
If we rename the fields too it will fit: https://gist.github.com/tompave-roo/d864848d473c4aea37df265183868461